woocommerce/woocommerce
# Retrieve/update product stock threshold via REST API #27291

### All Submissions:


### Changes proposed in this Pull Request:

Closes #27291.

### How to test the changes in this Pull Request:

1. Create a product, check "Manage Stock" and set a value in product stock threshold field.
2. Retrieve the product with v3 REST API, response should include low_stock_amount with your value.
3. PUT {"low_stock_amount":5}, response should include low_stock_amount with your new value.

### Other information:


### Changelog entry

Retrieve/update product stock threshold via REST API.
